grabbit/src/main/groovy/com/twcable/grabbit/resources/GrabbitResourceProvider.groovy

```diff
@@ -79,8 +79,10 @@ class GrabbitResourceProvider implements ResourceProvider {
             case ~/^\/grabbit\/content(\/)?$/:
                 log.debug "Resolving ${path} to ContentResource"
                 return new ContentResource(resolver, path)
+            case ~/^\/grabbit(\/)?$/:
```

Comment:
We should probably have our resources extend from the root resource, and set their respective super resource types to the root. 

Rather than make a new case, maybe we should just have the root resource returned as the default case below? The root provider path is /grabbit anyway.